code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1514,6 +1514,25 @@
 
 	M.adjustToxLoss(1)
 
+/datum/reagent/vomit
```

Comment:
Mixed reagents tend to have an additional effect which is a product of their mixture, not just of the combination of their elements. I just wonder if this isn't a very diluted HCl acid effect that you could avoid making a reagent for.

And we should typically avoid new reagents because they tend not to play well with anything. You can use vomit for stuff, sure, but then I suggest you be able to make it in the lab (by mixing stomach acid and nutriment) because a reagent that appears in a rather snowflakey way from a single event and doesn't really mix with anything else feels rather pointless.